core/src/value/primitive.rs

```diff
@@ -604,25 +604,19 @@ impl PrimitiveValue {
         }
     }
 
-    /// Getter method to provide a clean version of a string or multiple strings,
-    /// removing unwanted whitespaces
+    /// Removes unwanted whitespaces from a string or multiple strings
+    ///
+    /// String values that have unwanted whitespaces
```

Comment:
This documentation looks incomplete. Please consider replicating the prose found in other methods such as `to_str`, and adapting it to this operation. A [doc-test](https://doc.rust-lang.org/rustdoc/documentation-tests.html) would be very nice to have too.